Celeste.Mod.mm/Mod/Core/CoreModuleSettings.cs

```diff
@@ -320,9 +320,13 @@ public class CoreModuleSettings : EverestModuleSettings {
         public ButtonBinding MenuPageDown { get; set; }
 
         [SettingSubHeader("MODOPTIONS_COREMODULE_DEBUGMODE_SUBHEADER")]
+        [SettingInGame(false)]
+        [DefaultButtonBinding(0, Keys.OemTilde)]
+        public ButtonBinding ToggleDebugConsole { get; set; }
+
         [SettingInGame(false)]
         [DefaultButtonBinding(0, Keys.OemPeriod)]
-        public ButtonBinding DebugConsole { get; set; }
+        public ButtonBinding OpenDebugConsole { get; set; }
```

Comment:
Renaming the option will get rid of people's existing bindings, we should just change the display name via dialog files instead.